packages/store-ui/src/atoms/Price/Price.tsx

```diff
@@ -0,0 +1,44 @@
+import type { HTMLAttributes, ElementType, ReactNode } from 'react'
+import React, { forwardRef } from 'react'
+
+export type PriceFormatter = (price: number, listing: boolean) => ReactNode
+
+export interface PriceProps
+  extends Omit<HTMLAttributes<HTMLSpanElement>, 'children'> {
+  as?: ElementType
+  testId?: string
+  value: number
+  formatter?: PriceFormatter
+  listing?: boolean
+}
+
+export const Price = forwardRef<Omit<HTMLSpanElement, 'children'>, PriceProps>(
+  function Price(
+    {
+      as: Component = 'span',
+      testId = 'store-price',
+      value,
+      formatter = (price) => price,
+      listing,
```

Comment:
Is there another type of price? I say this because of there is, I suggest changing listing:boolean to variant: listing | installments etc

Reply:
According to our current store framework we have many other scenarios. Are they already covered in here ? 

https://github.com/vtex-apps/product-price

Reply:
Also. Please add an example on how to style product summary and PDP prices. Thanks !!

Reply:
I'm thinking about that too @tlgimenes but I couldn't think of any scenarios, but I agree that if there are more scenarios, it should be a string instead of a boolean.

I would prefer a prop called `type` instead of variant, so as not to confuse people (theme-ui).

Reply:
In here we called of `state`, but maybe type can be a better name. We need to keep a consistency on these names

Reply:
Good point! I liked the API you proposed. I'm going to update it to match the kinds of prices we have on `product-price`.

Reply:
but variant will confuse people because the theme-ui @icazevedo

Reply:
This may happen, but I think it's better than having both `type` and `state`. We could use `form`, like `forma` in pt. But I think people may confuse it with `forms`.

Reply:
Other possible names are `version`, `genre`, or `model`. Do you think we should move this discussion to slack?

Reply:
I continue thinking `type` is the better option. But I'm open to change my opinion. Ah, and we mus have the discussion here. For OSS

Reply:
Are you only fearing for confusing theme-ui with our implementation @emersonlaurentino? If so, I think this isn't an issue since all theme-ui related code will eventually disappear from this repo

Reply:
Yes, @tlgimenes